_sass/sidebar.scss

```diff
@@ -1,10 +1,53 @@
+$link-left-padding: 0.5em;
+$indent-space: 1em;
+$vertical-line-offset: -0.3em;
```

Comment:
The latter two here appear unused - could we drop them?
The first seems to be used only once - perhaps inline that usage?